Snippets/trim2.sqlpromptsnippet

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
I like that naming... thanks for the suggestion.  I will refactor those 2 snippets to reflect the new naming.